### PR TITLE
Explore by group type

### PR DESCRIPTION
**Pull request by BradleySappington**

### CODE REVIEW

**Comment by BradleySappington** [comment]
@mfixstsci ready for review

File: jwql/website/apps/jwql/data_containers.py [hunk 1]
```diff
@@ -32,7 +32,6 @@
 import tempfile
 
 from astropy.io import fits
-from astropy.table import Table
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/data_containers.py:35]: Were we not using `Table` in this module @BradleySappington?

File: jwql/website/apps/jwql/views.py [hunk 2]
```diff
@@ -825,13 +829,14 @@ def explore_image(request, inst, file_root, filetype, rewrite=False):
...
                'available_ints': image_info_list['available_ints'],
+               'extensions': extensions,
                'base_url': get_base_url(),
                'form': form}
 
     return render(request, template, context)
 
 
-def explore_image_ajax(request, inst, file_root, filetype, scaling="log", low_lim=None, high_lim=None, rewrite=False):
+def explore_image_ajax(request, inst, file_root, filetype, scaling="log", low_lim=None, high_lim=None, ext_name="SCI", rewrite=False):
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/views.py:839]: Do we always expect there to be a science extension available?

**Review by mfixstsci — APPROVED** [review]
Hey @BradleySappington this looks good to me, thanks for the changes. Just have a couple of questions here and then we can merge.

File: jwql/website/apps/jwql/data_containers.py (hunk 1 above)
Comment (BradleySappington) [reply jwql/website/apps/jwql/data_containers.py:35]: Correct, my linter told me it was imported and unused.  Doing minor code cleanups along the way.  There are more cleanups/errors/bugs in this file that need to be done, but they affect the logic directly so I didn't want to put them in this code request, they should be done separately.

File: jwql/website/apps/jwql/views.py (hunk 2 above)
Comment (BradleySappington) [reply jwql/website/apps/jwql/views.py:839]: Spoke with @bhilbert4 who confirmed before adding this code.

**Review by bhilbert4 — APPROVED** [review]
Functionally this looked good on the test server.

**Pull request merged**